### DISCUSSION TITLE
Welcome to ZTE-MF79-usb-modem-Send-SMS Discussions!

### DISCUSSION THREAD

**Discussion by rkarimabadi** [rkarimabadi/ZTE-MF79-usb-modem-Send-SMS#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fengjiaqi927** [comment]
Share ideas.
I'm a newbie, I try to make a high performance mobile hotspot with ZTE-MF79 and router.
My router has an openwrt system installed, and I wrote some scripts to automatically access ZTE-MF79’s webpage, extract traffic usage, and automatically turn off ZTE-MF79’s networking function after the traffic reaches the set threshold. In addition, the script will send a message to my account every day, telling me about the data usage.
the link below is my project, Welcome to use and give valuable comments~
[https://github.com/fengjiaqi927/ZTE-MF79U-shell-scripts](url)

**Comment by abaav159-droid** [comment]
With God's Help

Tanks for your project

I updated the App.config with the password and the IP, and compiled the project, but when I run it I receive :
 
ZTEModemTest.exe +972546884707 ABATest
Unhandled exception. System.NullReferenceException: Object reference not set to an instance of an object.
   at ZTEModemTest.Program.Main(String[] args) in C:\ZTE-MF79-usb-modem-Send-SMS-main\ZTEModemTest\Program.cs:line 10

Can you help ?
